plugins/copr.py

```diff
@@ -79,6 +81,29 @@ class CoprCommand(dnf.cli.Command):
   copr search tests
     """)
 
+# Useful for forcing a distribution
+    def _chroot_config(self):
+        cp = self.read_config(self.base.conf, PLUGIN_CONF)
```

Comment:
File "/usr/lib/python3.4/site-packages/dnf-plugins/copr.py", line 86, in _chroot_config
    cp = self.read_config(self.base.conf, PLUGIN_CONF)
AttributeError: 'CoprCommand' object has no attribute 'read_config'